resources/wikia/modules/tracker.stub.js

```diff
@@ -48,6 +51,9 @@
 				// Generic enable
 				ENABLE: 'enable',
 
+				// Action or flow end
+				END: 'end',
```

Comment:
could we add here more descriptive name? if it's an action describing the flow, as mentioned in comment, could we name it `FLOW_END` or `EDIT_END` ?